src/components/utils/test/test_generator/generated_msg_version_test.cc

```diff
@@ -56,7 +58,7 @@ void get_version_from_file(int& first_version, int& second_version, int& third_v
     std::size_t isfound = str.find("interface name");
     if (isfound != std::string::npos) {
       std::size_t versionStart = str.find("\"", str.find("version="));
-      ASSERT_TRUE(versionStart != std::string::npos);  
+      ASSERT_TRUE(versionStart != std::string::npos);
```

Comment:
Sorry, these style issues were introduced in a previous PR, they are now fixed by #1534. They should be removed from this PR before merge